app/main.py

```diff
@@ -1,34 +1,64 @@
-class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+from typing import List, Tuple, Dict
+
 
+class Deck:
+    def __init__(self, row: int, column: int,
+                 is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
```

Comment:
There is an attribute `is_alive` in the `Deck` class that is not used anywhere in the code. According to the checklist, all attributes defined in the `__init__` method should be used. Please remove it if it's not necessary. (Checklist point 4)